<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>748</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_png">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>671</width>
      <height>591</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/images/back.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="sure_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>surelabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_kesilenKarpuz">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>221</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Kesilen Karpuz Say覺s覺 : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_kacanKarpuz">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>60</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Kacan Karpuz Say覺s覺: </string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
